examples/Firebase_ESP8266_LEDs/Firebase_ESP8266_Neopixel/Firebase_ESP8266_Neopixel.ino

```diff
@@ -25,36 +25,38 @@
 const int PIN=13;
 Adafruit_NeoPixel strip = Adafruit_NeoPixel(32, PIN, NEO_GRB + NEO_KHZ800);
 
-// TODO: Replace with your own credentials and keep these safe.
-Firebase fbase = Firebase("YOUR-PROJECT.firebaseio.com", "YOUR_AUTH_SECRET");
+// Set these to run example.
+#define FIREBASE_HOST "example.firebaseio.com"
+#define FIREBASE_AUTH "token_or_secret"
+#define WIFI_SSID "SSID"
+#define WIFI_PASSWORD "PASSWORD"
 
-void setup() {
-  Serial.begin(9600);
+Firebase fbase = Firebase(FIREBASE_HOST, FIREBASE_AUTH);
 
-  strip.begin();
-  strip.setBrightness(25); // 0 ... 255
-  strip.show(); // Initialize all pixels to 'off'
-
-  // Not connected, set the LEDs red
-  colorWipe(&strip, 0xFF0000, 50);
+void ConnectWifi(const String& ssid, const String& password = "") {
+  if (pass != "") {
+    WiFi.begin(ssid, password);
+  } else {
+    WiFi.begin(ssid);
+  }
 
-  // connect to wifi.
-  WiFi.begin("GoogleGuest", "");
   Serial.print("connecting");
-
-  int count = 0;
   while (WiFi.status() != WL_CONNECTED) {
-    // Draw rainbows while connecting
     Serial.print(".");
-    if (count < strip.numPixels()){
```

Comment:
This is fancy and nice but will have to go for these to be standardized. We could add callback lambdas to allow these to stay but it would complicate the examples a bit and might get too c++-y.